src/Models/QuoteItemOption.php

```diff
@@ -23,26 +23,20 @@ public function quote_item()
     protected function value(): Attribute
     {
         return Attribute::make(
-            get: fn (string $value) => match ($this->code) {
-                'info_buyRequest' => json_decode($value),
-                'attributes'      => json_decode($value),
-                'option_ids'      => explode(',', $value),
-                default           => (function () use ($value) {
-                    if (! $this->option) {
-                        return;
-                    }
+            get: function (string $value) {
+                $value = Eventy::filter('quote_item_option.value', $value, $this);
 
-                    if (in_array($this->option->type, ['drop_down', 'radio'])) {
-                        return config('rapidez.models.product_option_type_value')::find($value)->title;
-                    }
+                if (isset($this->option) && in_array($this->option->type, ['drop_down', 'radio'])) {
+                    $value = config('rapidez.models.product_option_type_value')::find($value)->title;
+                }
 
-                    if ($this->option->type == 'file') {
-                        return json_decode($value);
-                    }
+                if ($this->code === 'option_ids') {
+                    $value = explode(',', $value);
+                }
 
-                    return $value;
-                })()
-            })->shouldCache();
+                return json_decode($value) ?? $value;
```

Comment:
Just to be sure php won't complain about types
```suggestion
                return is_string($value) ?  (json_decode($value) ?? $value) : $value;
```